### ISSUE TITLE
Report button inconsistency

### ISSUE THREAD

**Issue by emiljensen2** [pajlads/DinkPlugin#409]
### Checklist

- [X] I'm reporting a problem with Dink
- [X] I've searched the [issues and pull requests](https://github.com/pajlads/Dinkplugin/issues?q=) for similar looking reports

### Describe your issue

When dink takes a screenshot of events it doesn't always show the date despite it being enabled in the base screenshot plugin.
I'm not quite sure what causes it
Collection log events seem to always have the date show if popup is enabled, I see some cases of date not showing and they don't have the popup enabled.
CA events also seem to capture date consistently (Though this is without the popup, so unsure if it's always true)
Levels, loot, kill count, quests is inconsistent from what I've seen.

Sorry for yet another report

### Screenshots

![image](https://github.com/pajlads/DinkPlugin/assets/55587410/6216129c-4a86-44dd-9d53-27979c7b8373)
![image](https://github.com/pajlads/DinkPlugin/assets/55587410/1aad8f94-a3b3-40fe-ba0a-6d6e008d8456)

### Runelite Logs

shouldn't be relevant

### Runelite Version

1.10.19

**Comment by pajlada** [comment]
Could you compare these with the screenshots that were saved on disk (and maybe post them too)?

**Comment by emiljensen2** [comment]
I've taken some levels screenshots since my rl doesn't do loot itself.
![image](https://github.com/pajlads/DinkPlugin/assets/55587410/8a741623-d32b-4bf1-af79-c7152b46bda7)
![Runecraft(84) 2024-01-16_22-24-18](https://github.com/pajlads/DinkPlugin/assets/55587410/5e1d1acc-83d2-469c-a7f7-a602731646cd)
![image](https://github.com/pajlads/DinkPlugin/assets/55587410/f32297d3-421d-4737-a0a6-e4f7518785b2)
![Mining(91) 2024-01-10_19-21-35](https://github.com/pajlads/DinkPlugin/assets/55587410/b1f9d0c1-6068-403e-bede-56a04bd1c1c4)
The mining screenshot is interesting, they appear to be taken a little time apart as my idle notification is visible on the dink screenshot, but not the local screenshot.

**Comment by pajlada** [comment]
We delay certain notifiers, sometimes to make sure we get all level ups that happened, sometimes to give time for the loot to appear in the chat or on the ground for the screenshot.

I suspect this is just a matter of us not using the screenshot plugin, but just happen to be taking screenshots at the same time as the screenshot plugin for certain events.

I guess it boils down to more of a feature request: Should we replicate the screenshot plugin's "Report button" functionality?

**Comment by emiljensen2** [comment]
Ah that makes sense, I thought at first you were using the screenshot plugin's functionality so to speak. but that makes sense.
So it's inconsistent because it sometimes captures on the same frame/tick(?), but it is not intended.

Yeah I guess it'd be a feature request then.

**Comment by iProdigy** [comment]
Yeah we would have to replicate https://github.com/runelite/runelite/blob/master/runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotOverlay.java to implement this feature

**Comment by Felanbird** [comment]
The current upstream implementation of the report button does not respect where the report button actually is, if the chat box is somewhere else, it will just float in it's normal spot. Keep that in mind whether or not we want to fix it ourselves, fix it for them, or wait for someone to fix it.

Discussed in the discord last week but I don't believe anyone bothered to open an issue.
